app/templates/briefs/not_is_supplier_eligible_for_brief_error.html

```diff
@@ -1,43 +1,27 @@
 {% extends "_base_page.html" %}
 
-{% block page_title %}Not eligible for brief – Digital Marketplace{% endblock %}
+{% block page_title %}Not eligible for opportunity – Digital Marketplace{% endblock %}
 
 {% block main_content %}
 
-{% if clarification_question %}
+{% with inhibited_action="ask a question about" if clarification_question else "apply for" %}
 <div class="grid-row">
   <div class="column-two-thirds">
     <header class="page-heading-smaller">
-      <h1>You can’t ask a question about this opportunity</h1>
+      <h1>You can’t {{ inhibited_action }} this opportunity</h1>
     </header>
-    {% if on_framework %}
-      <p>
-        You can’t ask a question about this opportunity because you don’t provide
-        {{ "this digital specialist role" if lot == 'digital-specialists' else "a service in this category" }}
-        at this location.
-      </p>
-    {% else %}
-      <p>You can’t ask a question about this opportunity because you’re not a {{ framework_name }} supplier.</p>
-    {% endif %}
+    <p>
+      You can’t {{ inhibited_action }} this opportunity because
+      {% if has_framework_service %}
+        you didn’t say you could provide
+        {{ "this specialist role" if has_framework_lot_service else "services in this category" }}
```

Comment:
Ah yes, I see what you mean. No, it hasn't done any checks about the role - it's _deduced_ that that is what the problem is from the fact that you have a valid service on the framework & lot, but still when you filter by _all_ the eligibility criteria, it's not showing up. And the role (I _think_) is the only remaining filtering criterion. I've not made this obvious and it's at least comment worthy.